src/content/docs/en/guides/integrations-guide/node.mdx

```diff
@@ -106,6 +107,28 @@ export default defineConfig({
 });
 ```
 
+### `experimentalDisableStreaming`
+
+<p>
+**Type:** `boolean` <br />
+<Since v="9.3.0" pkg="@astrojs/node" />
+</p>
+
+Allows disabling [HTML streaming](/en/guides/on-demand-rendering/#html-streaming). This is useful, for example, if there are constraints from your host:
```

Comment:
I didn't see "constraints from your host" in the original discussion as an advantage, and I'm not honestly sure what these might be. Should we be mentioning async components as a common use case for which these are an advantage? It would also be helpful if we knew exactly what problem this solves so people can be sure when NOT to do this: e.g. is this a thing that will help in some situations, but come at some other cost? We should help people decide if this is something they want/need.